README.md

```diff
@@ -169,7 +169,7 @@ Prettier for PHP supports the following options. We recommend that all users set
 | `tabWidth`         | `4`       | Same as in Prettier ([see prettier docs](https://prettier.io/docs/en/options.html#tab-width))                                                                                                                                                                                                                                                                                         |
 | `useTabs`          | `false`   | Same as in Prettier ([see prettier docs](https://prettier.io/docs/en/options.html#tabs))                                                                                                                                                                                                                                                                                              |
 | `singleQuote`      | `false`   | If set to `"true"`, strings that use double quotes but do not rely on the features they add, will be reformatted. Example: `"foo" -> 'foo'`, `"foo $bar" -> "foo $bar"`.                                                                                                                                                                                                              |
-| `trailingCommaPHP` | `"all"`   | If set to `"all"`, trailing commas will be added wherever possible. <br> If set to `"none"`, no trailing commas are printed.                                                                                                                                                                                                                                                          |
+| `trailingCommaPHP` | `true`   | If set to `true`, trailing commas will be added wherever possible. <br> If set to `false`, no trailing commas are printed.                                                                                                                                                                                                                                                          |
```

Comment:
I think will be great to accept as boolean and as string, because prettier support is as string

Reply:
Hmm, so you mean to keep it similar to Prettier for JS, right? If the options parser allows for mixed types, we could do this. Shall we do it in a separate PR though?

Reply:
Yes and yes